.github/workflows/ci.yml

```diff
@@ -139,3 +139,12 @@ jobs:
       - name: Rebuild Wasm and verify it hasn't changed
         if: success() && steps.source.outputs.modified == 'true'
         run: deno task --cwd ${{ matrix.module }} wasmbuild --check
+
+  typos:
+    name: "spell check"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: crate-ci/typos@master
```

Comment:
I think we should run this step as part of `lint` job

Reply:
sure